wasm/tests-js/src/message.pack-signed.test.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    MESSAGE_SIMPLE,
+    ExampleDIDResolver,
+    ALICE_DID_DOC,
+    PLAINTEXT_MSG_SIMPLE,
+    MESSAGE_MINIMAL,
+    PLAINTEXT_MSG_MINIMAL, ExampleSecretsResolver, ALICE_SECRETS, ALICE_DID,
+} from "./test-vectors";
+
+import {PackSignedMetadata} from "didcomm-js";
+
+test.each([
+    {
+        message: MESSAGE_SIMPLE,
+        sign_by: ALICE_DID,
+        sign_by_kid: "did:example:alice#key-3",
```

Comment:
> "did:example:alice#key-3"

Would be nice to keep `ALICE_AUTH_METHOD_25519.id` as in the rust tests

Reply:
fixed